playbooks/site.yml

```diff
@@ -12,42 +12,5 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Configure Kubernetes VIP
-  hosts: "{{ kubernetes_control_plane_group | default('controllers') }}"
-  become: true
-  roles:
-    - role: keepalived
-      tags:
-        - keepalived
-
-    - role: haproxy
-      tags:
-        - haproxy
-
-- name: Install Kubernetes
-  hosts: "{{ kubernetes_group | default('all') }}"
-  become: true
-  roles:
-    - role: containerd
```

Comment:
I'd change this to say `vexxhost.containers.containerd` instead of `containerd`.